lib/index.js

```diff
@@ -160,7 +160,7 @@ function extendTemplateCompiler(obj) {
   obj.compile = compileTemplate;
 }
 
-const transpile = renderCompiler; // enhance vue-template-compiler with babel:
+const transpile = {}; // enhance vue-template-compiler with babel:
```

Comment:
`transpile()` or `renderCompiler()`  is useful, not reduant, 

It can be used to replace [vue-template-es2015-compiler](https://github.com/vuejs/vue-template-es2015-compiler) by run `sh ./useVueTemplateBabelCompiler.sh`

You can refer to [previous README for Usage details](https://github.com/JuniorTour/vue-template-babel-compiler/commit/9317f827695a809189e5b3a822ca8f41c3ea0f93#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L35).

I will add this **workaround usage** laterly.